### PR TITLE
Fetch mangas during the update

### PR DESCRIPTION
**Pull request by schroda**
The update only fetched the chapter list of a manga but never the manga itself. Thus, e.g. unless the manga got online fetched via the ui, it would never get recognized if it is completed or not. This would e.g. prevent the update setting, to not update completed mangas, from working as intended

### CODE REVIEW

**Comment by Syer10** [comment]
Can you make this an option/parameter of the endpoint? In Tachiomi this is a optional setting that is off by default.

**Comment by schroda** [comment]
added a setting

**Review by Syer10 — APPROVED** [review]

**Comment by Syer10** [comment]
Ill merge this after #738 is merged and there is a graphql setting for it.

**Comment by schroda** [comment]
done

**Pull request merged**
